package.json

```diff
@@ -77,6 +77,7 @@
     "sha3": "1.2.3",
     "bignumber.js": "9.0.0",
     "prettier": "^1.19.1",
-    "@types/prettier": "^1.19.0"
+    "@types/prettier": "^1.19.0",
+    "ganache-core": "https://github.com/celo-org/ganache-core.git#3dbc32b"
```

Comment:
I have a slight preference for using the github link inline where we declare a dep on ganache-core instead of overriding here at the root. It might confused devs who try to upgrade it. Consider using this link throughout the monorepo or moving the ganache-core dep up here to the root like we did for typescript

Reply:
Yes I'd prefer that too, but `ganache-core` is pulled by `@0x/subproviders`, so we'd need to fork that too.

```txt
❯ yarn why "ganache-core"
=> Found "ganache-core@2.7.0"
info Reasons this module exists
   - "_project_#@celo#protocol#@0x#subproviders" depends on it
   - Hoisted from "_project_#@celo#protocol#@0x#subproviders#ganache-core"
```

Reply:
Ok thanks for clarifying